spark-3.1-spanner-lib/src/test/java/com/google/cloud/spark/spanner/SpannerTestBase.java

```diff
@@ -42,34 +42,59 @@
 import org.apache.spark.sql.catalyst.expressions.GenericInternalRow;
 import org.apache.spark.sql.catalyst.util.GenericArrayData;
 import org.apache.spark.unsafe.types.UTF8String;
-import org.junit.AfterClass;
 import org.junit.BeforeClass;
 
 class SpannerTestBase {
-  // TODO: Generate dynamic database name to avoid conflicts.
   private static String databaseId = System.getenv("SPANNER_DATABASE_ID");
-  private static String instanceId = System.getenv("SPANNER_INSTANCE_ID");
+  private static String instanceId = System.getenv("SPANNER_INSTANCE_ID") + "-" + System.nanoTime();
```

Comment:
Looks like 103 is having nanoTime after database id. Can we merge 103 then update the pull request ?

Reply:
Done, thanks!